### PR TITLE
Update screenlogic.markdown

### PR DESCRIPTION
**Pull request by dieselrabbit**
## Proposed change

Update service calls: `target` -> `config_entry`

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/109041
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
Thanks, @dieselrabbit 👍 

../Frenck

File: source/_integrations/screenlogic.markdown [hunk 1]
```diff
@@ -42,27 +42,27 @@ ScreenLogic options are set via **Settings** -> **Devices & Services** -> **Pent
...
 ### `screenlogic.stop_super_chlorination`
 
 Stops super chlorination.
 
-| Service data attribute | Optional | Description                                                                                                                                                       |
-| ---------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `target`               | no       | An `area` containing the ScreenLogic device, the ScreenLogic `device` itself, or any `entity` from the ScreenLogic device you wish to stop super chlorination on. |
+| Service data attribute | Optional | Description                                                                          |
+| ---------------------- | -------- | ------------------------------------------------------------------------------------ |
+| `config_entry`         | no       | Integration entry_id of the ScreenLogic instance wish to stop super chlorination on. |
```
Comment (MartinHjelmare) [inline source/_integrations/screenlogic.markdown]:
```suggestion
| `config_entry`         | no       | Integration entry_id of the ScreenLogic instance you wish to stop super chlorination on. |
```

**Comment by frenck** [comment]
Something is wrong here... it didn't allow me to merge, and now there is a merge conflict...

**Comment by MartinHjelmare** [comment]
The PR without my suggestion is already committed to the next branch.

https://github.com/home-assistant/home-assistant.io/commit/d49144b66cbf82690c45174ed57296dd3b11383e

**Comment by frenck** [comment]
> The PR without my suggestion is already committed to the next branch.
> 
> [d49144b](https://github.com/home-assistant/home-assistant.io/commit/d49144b66cbf82690c45174ed57296dd3b11383e)

Yes, that is the odd thing... GitHub gave an error during merge. So, it seems merged, but messed up the link to the PR or something...

anyways, merged next into it, with your change. Let's try again!

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
CleanShot 2024-02-05 at 09 38 44@2x

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**